service/src/test/java/com/codeforcommunity/processor/ProtectedUserProcessorImplTest.java

```diff
@@ -31,57 +35,88 @@
     /**
      * Method to setup mockDb and processor.
      */
+    @BeforeEach
     void setup() {
         this.mockDb = new JooqMock();
         this.processor = new ProtectedUserProcessorImpl(mockDb.getContext());
     }
 
-    // successfully deletes user
+    // successfully deletes user when user is a member
     @Test
     public void testDeleteUser1() {
-        setup();
-        UsersRecord myUser = mockDb.getContext().newRecord(Tables.USERS);
-        myUser.setUsername("kiminusername");
-        myUser.setEmail("kimin@example.com");
-        myUser.setPassHash(Passwords.createHash("password"));
-        myUser.setId(1);
-        myUser.setPrivilegeLevel(PrivilegeLevel.STANDARD);
-        mockDb.addReturn("SELECT", myUser);
-        mockDb.addReturn("DELETE", myUser);
+        UserTeamRecord myUserTeam = mockDb.getContext().newRecord(Tables.USER_TEAM);
+        myUserTeam.setUserId(1);
+        myUserTeam.setTeamId(5);
+        myUserTeam.setTeamRole(TeamRole.MEMBER);
+        mockDb.addReturn("SELECT", myUserTeam);
+        mockDb.addEmptyReturn("DELETE");
 
         JWTData jwtData = new JWTData(1, PrivilegeLevel.STANDARD);
 
         this.processor.deleteUser(jwtData);
         assertEquals(1, mockDb.timesCalled("SELECT"));
         assertEquals(3, mockDb.timesCalled("DELETE"));
+        assertEquals(1, mockDb.getSqlBindings().get("DELETE").get(0)[0]);
+        assertEquals(1, mockDb.getSqlBindings().get("DELETE").get(1)[0]);
+        assertEquals(1, mockDb.getSqlBindings().get("DELETE").get(2)[0]);
+        assertEquals(1, mockDb.getSqlBindings().get("SELECT").get(0)[0]);
```

Comment:
Could you go back and add little descriptions for the field that you're getting when you're testing with `getSqlBindings()`? Something like 
```
// userId
```
would be perfect

Reply:
done